sample/src/androidTest/java/com/schibsted/spain/barista/sample/RecyclerViewTest.java

```diff
@@ -26,4 +27,10 @@ public void checkClickRecyclerViewItem_byPosition_atThree() {
     clickRecyclerViewItem(R.id.recycler, 3);
     assertDisplayed("Raspberry");
   }
+
+  @Test
+  public void checkScrollToRecyclerViewItem_byPosition_atThree() {
+    scrollTo(R.id.recycler, 3);
```

Comment:
Is this test really testing that the scrollTo works? I mean, as the Recycler has only three items, the assertion will pass also if we don't call `scrollTo()`, isn't it?